scripts/mock-ols-server.mjs

```
@@ -0,0 +1,1034 @@
+  if (value === undefined || value === '') {
+    return fallback;
+  }
+  const parsed = Number(value);
+  return Number.isFinite(parsed) && parsed >= 0 ? parsed : fallback;
+};
+
+const DEFAULT_LIST_LIMIT = parseListLimit(process.env.MOCK_LIST_LIMIT, 4);
+const POD_LIST_LIMIT = parseListLimit(process.env.MOCK_POD_LIMIT, 0);
+const MOCK_DIVERSE_POD_STATUSES =
+  process.env.MOCK_DIVERSE_POD_STATUSES === '1' ||
+  process.env.MOCK_DIVERSE_POD_STATUSES === 'true';
```

ESLint

[error] Delete `⏎·`

(prettier/prettier)